app/src/main/java/com/sdp/movemeet/UploadActivityActivity.java

```diff
@@ -361,7 +358,7 @@ private Activity validateActivity() {
 
         String organizerId = FirebaseAuth.getInstance().getCurrentUser().getUid();
         Activity activity = new Activity(
-                organizerId + " || " +  date, organizerId, title, nParticipants,
+                organizerId + " || " + date, organizerId, title, nParticipants,
```

Comment:
We should change this way of defining an activity ID. Perhaps we should have a hashing function which hashes both the organizer, sport and creation date.

Reply:
yep, good idea. And maybe we can call it directly inside the Activity constructor